#name = "Harry"
# print(type(name))
#greeting = "Good Morning, "
#print(greeting + name)

# name = "Harry"
# print(name[3])
# name[3] = "d" --->Does not work

# print(name[0:3])
# print(name[:3])
# print(name[1:])

# c = name[-4:-1]
# print(c)

# c = name[-4:-1]
# print(c)
# d = name[1:4:2]
# print(d)

# name = "HarryIsGood"
# d = name[0::2]
# print(d)
# story = "Once upon a time there was a youtuber named Harry who uploaded python course with notes"

# #String Functions
# # print(len(story))
# # print(story.endswith("sdrrrrrr"))
# # print(story.endswith("notes"))
# # print(story.count("course"))
# # print(story.capitalize())

# # print(story.find("Harry"))
# print(story.replace("Harry","CodeWithHarry"))


# story = "Harry is good.\nHe is very good"
# print(story)


# name = input("Enter your  name: ")
# print("Good Morning "+name)

# name = input("Enter your name : ")
# Date = input("Enter the Date: ")

# templete ='''

# Dear <!name>
# you are selected !
# <!Date!>

# '''
# print(templete.replace('<!name>',name).replace('<!Date!>',Date))

# mystr = "This is me  and  I am a good boy"
# print(mystr.find("  "))

# letter = "Dear Harry,\n\tThis python course is nice.\nThanks"
# print(letter)
